docs/src/quickstart.md

```diff
@@ -0,0 +1,158 @@
+# Quickstart
+
+Welcome to the **Quickstart** guide for [`HealthBase.jl`](https://github.com/JuliaHealth/HealthBase.jl)!  
+This guide walks you through setting up your Julia environment, creating example OMOP CDM data, validating it, and applying preprocessing steps using the `HealthTable` system.
+
+## Getting Started
+
+### Launch Julia and Enter Your Project Environment
+
+To get started:
+
+1. Open your terminal or Julia REPL.
+2. Navigate to your project folder (where `Project.toml` is located):
+
+```sh
+cd path/to/your/project
+```
+
+3. Activate the project:
+
+```julia
+julia --project=.
+```
+
+4. (Optional for docs) For working on documentation:
+
+```sh
+julia --project=docs
+```
+
+## 1. Load Packages
+
+Before loading `HealthBase`, you must first load some **trigger packages**.  
+These packages enable HealthBase's extensions, which power important features like type validation and concept mapping.
+
+> ⚠️ **Important:** Load the following packages **before** `using HealthBase`.  
+> Otherwise, some functions may not be available due to missing extensions.
+
+```julia
+# First, load the trigger packages
+using DataFrames, OMOPCommonDataModel, InlineStrings, Serialization, Statistics, Dates, FeatureTransforms, DBInterface, DuckDB
+
+# Then, load HealthBase
+using HealthBase
+```
+
+## 2. Create Example DataFrames
+
+We'll create two `DataFrame`s:
+
+* `good_df` - a minimal, valid slice of the OMOP *person* table.
+* `wrong_df` - intentionally invalid (wrong types & extra column) so you can see the constructor’s validation in action.
+
+```julia
+good_df = DataFrame(
+    person_id = 1:6,
+    gender_concept_id = [8507, 8507, 8532, 8532, 8507, 8532],
+    year_of_birth = [1980, 1995, 1990, 1975, 1988, 2001],
+    race_concept_id = [8527, 8515, 8527, 8516, 8527, 8516]
+)
+
+# Invalid DataFrame to test validation
+wrong_df = DataFrame(
+    person_id = ["1", "2"],            # Should be Int64
+    gender_concept_id = [8507, 8532],
+    year_of_birth = [1990, 1985],
+    race_concept_id = [8527, 8516],
+    illegal_extra_col = [true, false],  # Invalid column
```

Comment:
Indu, if you could suppress the error/warning message about extra columns that are not part of the OMOP CDM but are valid DataFrame columns, that would be perfect. That way, HealthTable users don't think they can't add additional columns to the DataFrame. Otherwise, this looks great! After thinking some more about this, maybe just also label the "illegal_extra_col" as "extra_col" in the quickstart as well.